src/main/java/VisualLogic/Element.java

```diff
@@ -2499,17 +2474,17 @@ public void writePinObj(int index, Object object) {
 
     @Override
     public void writePinBoolean(int index, boolean value) {
-        writeInformationToDest(index, new Boolean(value));
+        writeInformationToDest(index, value);
     }
 
     @Override
     public void writePinInteger(int index, int value) {
-        writeInformationToDest(index, new Integer(value));
+        writeInformationToDest(index, Integer.valueOf(value));
```

Comment:
I think I've automated the process maybe too much, also invited by IDE. IntelliJ thinks that wrappers automatically converts primitive types into wrappers.
I'll restore to .valueOf, because writeInformationToDest requires Object in some cases.

Reply:
Primitives are automatically boxed using `Integer#valueOf` simply passing the primitive value should work.